.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ci
+
+on:
+  pull_request:
+    branches:
+      # on pull requests to develop and release branches
+      - develop
```

Comment:
are we going to use develop or will be switching to main? because once we start releasing, IMO main should be used.

Reply:
we will never "release" from main -- it is going to be a development branch. Actual releases will be done from tagged branches, hence this convention.